packages/beacon-node/src/chain/validation/blobsBundle.ts

```diff
@@ -0,0 +1,41 @@
+import {BlobsBundle} from "../../execution/index.js";
+import {ckzg} from "../../util/kzg.js";
+import {CELLS_PER_EXT_BLOB, ForkName, ForkSeq} from "@lodestar/params";
+
+/**
+ * Validate blobsBundle according to spec
+ * https://github.com/ethereum/execution-apis/blob/cad4194e3fa37359a1be95e4aad2752d69691077/src/engine/osaka.md#specification
+ */
+export function validateBlobsBundle(fork: ForkName, blobsBundle: BlobsBundle): void {
+  if (blobsBundle.commitments.length !== blobsBundle.blobs.length) {
+    throw Error(
+      `Invalid BlobsBundle: commitments.length ${blobsBundle.commitments.length} != blobs.length ${blobsBundle.blobs.length}`
+    );
+  }
+
+  if (ForkSeq[fork] >= ForkSeq.fulu) {
```

Comment:
i see this is already forkaware so better integrate it back to validateBlobsAndKzgCommitments or merge it here because both are doing the same thing